packages/ocean-react/src/CrossSellCard/CrossSellCard.tsx

```diff
@@ -54,13 +39,20 @@ const CrossSellCard = React.forwardRef<HTMLDivElement, CrossSellCardProps>(
           className="ods-cross-sell-card__content"
           onClick={() => ctaAction}
           data-testid="cta-test"
+          style={{ background: backgroundColor || '#0025e0' }}
```

Comment:
As the ocean-web follows a design system it can’t receive colors that don't exist in the design system. So I suggest consulting with @julianasouza-s  first.

Reply:
We are still testing a few background colors, including gradients, but all of them uses (well, it should use) colors that are part of our design system.